tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php

```diff
@@ -28,6 +28,9 @@ class LocalFooterLockupCest {
   function _before(AcceptanceTester $I) {
     $this->DATA_DIR = rtrim(codecept_data_dir(), '/\\');
     // Copy our assets into place first.
+    if (!file_exists($this->DATA_DIR . DIRECTORY_SEPARATOR)) {
```

Comment:
Shouldn't this be `is_dir`?

Reply:
https://www.php.net/manual/en/function.file-exists.php
> Checks whether a file or directory exists.

Tomato Potato.

Reply:
WOrDs MeAN THingS!